daemon.c

```diff
@@ -673,6 +675,25 @@ static void register_nfs_service(SVCXPRT * udptransp, SVCXPRT * tcptransp)
 		    "unable to register (NFS3_PROGRAM, NFS_V3, udp).");
 	    daemon_exit(0);
 	}
+
+	if (opt_portmapper) {
+	    // FIXME: do not register if using only ipv4
```

Comment:
Koukám, že to FIXME řešíte v nějakém dalším commitu. Buďto napište do commit message, že to vyřešíte v dalším comitu, ale lepší by bylo, když to vyřešíte rovnou v tomto commitu.